lib/Dialect/Indexing/IR/Indexing.cpp

```diff
@@ -0,0 +1,74 @@
+//===-- Indexing.cpp - Indexing dialect -----------------------*- C++ -*-===//
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "structured/Dialect/Indexing/IR/Indexing.h"
+
+#include "mlir/Dialect/Tensor/IR/Tensor.h"
+#include "mlir/IR/DialectImplementation.h"
+#include "mlir/IR/TypeUtilities.h"
+#include "mlir/Support/LogicalResult.h"
+#include "mlir/Transforms/InliningUtils.h"
+#include "llvm/ADT/ScopeExit.h"
+#include "llvm/ADT/TypeSwitch.h"
+
+using namespace mlir;
+using namespace mlir::structured;
+
+//===----------------------------------------------------------------------===//
+// Indexing dialect
+//===----------------------------------------------------------------------===//
+
+#include "structured/Dialect/Indexing/IR/IndexingOpsDialect.cpp.inc"
+
+void IndexingDialect::initialize() {
+#define GET_OP_LIST
+  addOperations<
+
+#include "structured/Dialect/Indexing/IR/IndexingOps.cpp.inc"
```

Comment:
I think people usually don't leave empty lines but I only care for consistency purposes...